testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -261,11 +270,13 @@ Feature: Synchronize products in the products page of the Setup Wizard
     When I click the Add Product button
     And I wait until I see "Selected channels/products were scheduled successfully for syncing." text
     And I wait until I see "SUSE Linux Enterprise Micro 5.2 x86_64" product has been added
+    And I wait until all synchronized channels for "suse-microos-5.2" have finished
```

Comment:
I don't see `suse-microos-5.2` defined in CHANNEL_TO_SYNCH_BY_OS_PRODUCT_VERSION

Reply:
it's there
```
  'suse-microos-5.2' =>
    %w[
```